src/runtime_src/core/edge/skd/CMakeLists.txt

```diff
@@ -33,6 +33,7 @@ target_link_libraries(skd
   xrt_core
   xrt_coreutil
   dl
+  stdc++fs
```

Comment:
First let me say that I haven't checked, but does this now mean that this code base is:
1. Compiled with C++17 or greater (by using this library I must assume so).
2. That all boost::filesystem calls have been removed.

If the code is compiled with C++20, then lets do talk about replacing boost::format with std::format.